src/composable/chat/useChatHighlights.ts

```diff
@@ -222,28 +279,55 @@ export function useChatHighlights(ctx: ChannelContext) {
 		return toReactive(data.highlights);
 	}
 
+	function getAllIgnored(): Record<string, IgnoreDef> {
```

Comment:
Should just use `getAll()` since you pass a prop to the composable to determine the mode